python/NLP/subscriber.py

```diff
@@ -0,0 +1,101 @@
+# everything that is part of the standard library
+import json
+import asyncio
+import warnings
+
+# everything that has to get pip or conda installed
+import spacy
+import msgpack
+from bs4 import BeautifulSoup
+from textblob import TextBlob
+from pyensign.ensign import Ensign
+from pyensign.api.v1beta1.ensign_pb2 import Nack
+
+# local package components
+from config import ENSIGN_CLIENT_ID, ENSIGN_CLIENT_SECRET
```

Comment:
I have a similar comment in the other PR, but I don't think this will work without the `config.py` file, so we should either commit it with empty strings for the secrets or rely on the environment variables instead.